mobile/docs/root/api/starting_envoy.rst

```diff
@@ -527,6 +527,60 @@ Note that Envoy will fail to start up in debug mode if an unknown guard is speci
   // Swift
   builder.setRuntimeGuard("feature", true)
 
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+``addRtdsLayer``
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Adds an RTDS layer to the bootstrap configuration.
+Requires that ADS be configured via `setAggregatedDiscoveryService()`.
+See the following link for details:
+https://www.envoyproxy.io/docs/envoy/latest/api-v3/service/runtime/v3/rtds.proto
+
+**Example**::
+
+  // Kotlin
```

Comment:
Nit: Should examples be added for Swift & C++ too? When the syntax is valid for more than one language we should also just do something like `// C++ / Kotlin / Swift` and have a single code sample instead of duplicating for each language.